lib/spack/llnl/util/cpu/microarchitectures.json

```diff
@@ -1012,6 +1012,7 @@
         },
         "clang": {
           "versions": "3.9:",
+          "name": "pwr7",
```

Comment:
As far as I can tell, all of these microarchitectures are just as applicable to the non-LE PPC64 architecture, so I've supplied them here.